### PR TITLE
OWNERS: Add lyarwood and 0xFelix as reviewers

### PR DESCRIPTION
**Pull request by lyarwood**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by lyarwood** [comment]
/hold

Asking for additional volunteers before merging this.

**Review by 0xFelix — APPROVED** [review]
/lgtm

**Pull request merged**
